src/plugins/directory.js

```diff
@@ -0,0 +1,59 @@
+export const classProperties = {
```

Comment:
Oh yeah, sorry, I was commenting with just a fuzzy recollection of how my own PR worked. So `transform` vs `syntax` is accounted for, by way of both being mentioned in the message.

And now I see your edit above. So yeah, that's a way to encode the same information less explicitly. However I would still kinda prefer to move this out of Babylon if possible, as discussed below.